shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/loader/model/SchemaMetaData.java

```diff
@@ -25,8 +25,8 @@
 import java.util.Collection;
 
 /**
- * Schema meta data.
```

Comment:
Why change the javadoc style?

Reply:
No anything change in my local env. Let me check it